gobblin-dataset-config/src/main/java/gobblin/config/client/ConfigClient.java

```diff
@@ -0,0 +1,17 @@
+package gobblin.config.client;
+
+import java.net.URI;
+
+import com.typesafe.config.Config;
+
+import gobblin.config.configstore.ConfigStore;
+
+public interface ConfigClient {
```

Comment:
gobblin.dataset.config.ConfigClient is the deprecated one and will not be used. 

This file is used for client to get configs